contracts/Governance/GovernorBravo.sol

```diff
@@ -0,0 +1,364 @@
+      * @param description String description of the proposal
+      * @return Proposal id of new proposal
+      */
+    function propose(address[] memory targets, uint[] memory values, string[] memory signatures, bytes[] memory calldatas, string memory description) public returns (uint) {
+        // Reject proposals before initiating as Governor
+        require(PUSH.getPriorVotes(msg.sender, block.number - 1) > proposalThreshold, "GovernorBravo::propose: proposer votes below proposal threshold");
+        require(targets.length == values.length && targets.length == signatures.length && targets.length == calldatas.length, "GovernorBravo::propose: proposal function information arity mismatch");
+        require(targets.length != 0, "GovernorBravo::propose: must provide actions");
+        require(targets.length <= proposalMaxOperations, "GovernorBravo::propose: too many actions");
+
+        uint latestProposalId = latestProposalIds[msg.sender];
+        if (latestProposalId != 0) {
+          ProposalState proposersLatestProposalState = state(latestProposalId);
+          require(proposersLatestProposalState != ProposalState.Active, "GovernorBravo::propose: one live proposal per proposer, found an already active proposal");
+          require(proposersLatestProposalState != ProposalState.Pending, "GovernorBravo::propose: one live proposal per proposer, found an already pending proposal");
+        }
+
+        uint startBlock =  block.number - votingDelay;
+        uint endBlock = startBlock - votingPeriod;
+
+        proposalCount++;
+        uint newProposalID = proposalCount;
+        Proposal storage newProposal = proposals[newProposalID];
+
+            newProposal.id= proposalCount;
+            newProposal.proposer= msg.sender;
+            newProposal.eta= 0;
+            newProposal.targets= targets;
+            newProposal.values= values;
+            newProposal.signatures= signatures;
+            newProposal.calldatas= calldatas;
+            newProposal.startBlock= startBlock;
+            newProposal.endBlock= endBlock;
+            newProposal.forVotes= 0;
+            newProposal.againstVotes= 0;
+            newProposal.abstainVotes= 0;
+            newProposal.canceled= false;
+            newProposal.executed= false;
+        
+        latestProposalIds[newProposal.proposer] = newProposal.id;
+
+        emit ProposalCreated(newProposal.id, msg.sender, targets, values, signatures, calldatas, startBlock, endBlock, description);
+        return newProposal.id;
+    }
+
+    /**
+      * @notice Queues a proposal of state succeeded
+      * @param proposalId The id of the proposal to queue
+      */
+    function queue(uint proposalId) external {
+        require(state(proposalId) == ProposalState.Succeeded, "GovernorBravo::queue: proposal can only be queued if it is succeeded");
+        Proposal storage proposal = proposals[proposalId];
+        uint eta = block.timestamp - timelock.delay();
```

Comment:
Line 125: Critical Issue

Used subtraction instead of addition. Disturbs the intended logic of the function.